apps/communications/pbap/js/pbap.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+(function (exports){
+  var managerBt = navigator.mozBluetooth;
+  var managerContacts = navigator.mozContacts;
+  var adapter;
+  var pb;
+
+  function pbapInit(){
+    pb = new PbapPhonebook();
+    if (!managerBt) {
+      console.log('PBAP', 'No Bluetooth Manager');
+      return;
+    }
+    var req = managerBt.getDefaultAdapter();
+    req.onsuccess = function bt_getAdapterSuccess() {
+      console.log('PBAP', 'adapter success');
+      adapter = req.result;
+      registerPbapEvent();
```

Comment:
There is no need to have a separate method for this, just inline the code here, also please use addEventListener() to register events, it's preferred